app/assets/stylesheets/darkswarm/shop.css.scss

```diff
@@ -11,10 +11,145 @@
 @import "shop-taxon-flag";
 @import "shop-popovers";
 
+$sidebar-small-width: 85%;
+$sidebar-medium-width: 65%;
+$sidebar-large-width: 45%;
+$sidebar-footer-height: 5em;
+
 .darkswarm {
+  .shop-searchbar {
+    background-color: $grey-100;
+    height: 5em;
+    padding: 1em 0;
+    margin-bottom: 1em;
+    position: relative;
+    z-index: 5;
+
+    input#search {
+      height: 3em;
+      border-radius: 0.25em;
+      border: solid 1px $grey-300;
+      margin: 0;
+      padding: 0 1em 0 2.75em;
+      width: 100%;
+      min-width: 0;
+      background: $white url("/assets/icn-search-grey.png") 1em center no-repeat;
+      font-size: 1em;
+
+      &::placeholder {
+        font-style: italic;
+      }
+    }
+
+    button {
+      background-color: $grey-600;
+      margin-left: 1em;
+      height: 3em;
+      width: 7em;
+      padding: 0;
+      font-size: 1em;
+
+      @media all and (max-width: 480px) {
+        margin-left: 0.75em;
+      }
+    }
+
+    @media all and (max-width: 480px) {
```

Comment:
can we extract this breakpoint in a var so we remove this duplication?

Reply:
Done :+1: